packages/helix_editor.rb

```diff
@@ -0,0 +1,78 @@
+require 'package'
+
+class Helix_editor < Package
+  description 'A vim inspired editor with LSP support'
+  homepage 'https://helix-editor.com/'
+  version '22.12'
+  license 'MPL-2.0' # license of source
+  compatibility 'all'
+  source_url 'https://github.com/helix-editor/helix/archive/22.12.tar.gz'
+  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972' # Use the command "sha256sum"
+
+  depends_on 'rust' => :build
+  depends_on 'xdg_base'
+
+  @no_fhs = true
```

Comment:
Only need `no_fhs`?

Reply:
Maybe library shrinking / stripping and patchelf should be disabled?
The libraries are standalone and compiled during the build process, so patchelf is not useful I guess.
I don't really understand what the shrinking/stripping is exactly.

Reply:
You can specify `no_shrink` or `no_strip`.  See `crew help prop no_shrink` or `crew help prop no_strip` for more information.  To display a list of all available properties, enter `crew prop`.

Reply:
ok. I added no_shrink and no_strip. I also added no_patchelf. Is patchelf required for libraries that are build with crew?

Reply:
Just a note in case you didn't catch it.  All these properties do not have to be explicitly set to true,  You could also add them like below:
```
no_fhs
no_shrink
no_strip
no_patchelf
```

Reply:
The `no_patchelf` can be used to bypass this process and is optional.

Reply:
Good to know. I am not familiar with ruby. I removed the explicit assignment.
I still have no clue if I should set no_patchelf or not.
```
readelf -d hx | grep -i needed
 0x00000001 (NEEDED)                     Shared library: [libgcc_s.so.1]
 0x00000001 (NEEDED)                     Shared library: [libm.so.6]
 0x00000001 (NEEDED)                     Shared library: [libc.so.6]
 0x00000001 (NEEDED)                     Shared library: [ld-linux-armhf.so.3]
```

Reply:
Any ideas @satmandu?